src/main/java/org/jabref/logic/bibtex/FieldContentParser.java

```diff
@@ -28,6 +28,7 @@ public FieldContentParser(FieldContentParserPreferences prefs) {
         multiLineFields = new HashSet<>();
         // the following two are also coded in org.jabref.logic.bibtex.LatexFieldFormatter.format(String, String)
         multiLineFields.add(FieldName.ABSTRACT);
```

Comment:
Please extract these fields that should be treated as multilines to `InternalBibtexFields` (or a better location). Moreover, a few places still work with https://github.com/JabRef/jabref/blob/733da312471c3abab691d61eccee061962357e1c/src/main/java/org/jabref/model/entry/FieldName.java#L107 instead of `comments`. This should be changed too.